requirements/freeze.pip

```diff
@@ -5,7 +5,6 @@ django-filter==1.0.4
 django-jchart==0.4.1
 Django-Select2==5.11.1
 gunicorn==19.7.1
-psycopg2==2.7.3.1
```

Comment:
As a result of this you should add `- pip install psycopg2` in `.travis.yml` file in the `before_install` section.